backend/battle/tests/test_view.py

```diff
@@ -1,9 +1,113 @@
-from django.test import TestCase
+from django.test import TestCase, Client
 from model_bakery import baker
-from battle.models import PokemonTeam
+from django.urls import reverse
+from users.models import User
+from battle.models import Battle, PokemonTeam
 from battle.battles.battle import get_winner_for
 
 
+class ListBattlesTest(TestCase):
+    def setUp(self):
+        self.client = Client()
+        self.battle = baker.make('battle.Battle')
+        self.user = User.objects.create(email='example@example.com')
+        self.user.set_password('admin')
+        self.user.save()
+
+    def test_login_user_can_acess_battle_list(self):
+        self.client.login(username=self.user.email, password='admin')
+        response = self.client.get(reverse('battle_list'))
+        self.assertEqual(response.status_code, 200)
+
+    def test_data_returns_empty_list(self):
```

Comment:
I think the naming of the tests here could be improved, the `data` is not making much sense. Maybe `test_view_returns...`

Reply:
Alright :)